fluent-plugin-events/lib/sumologic/kubernetes/connector.rb

```diff
@@ -7,17 +7,41 @@ module Connector
       K8_POD_CA_CERT = 'ca.crt'.freeze
       K8_POD_TOKEN = 'token'.freeze
 
-      def connect_kubernetes
-        @client = Kubeclient::Client.new(
-          @kubernetes_url, @api_version,
+      # Need different clients to access different API groups/versions
+      CORE_API_VERSIONS = ['v1'].freeze
+      API_GROUPS = ['events.k8s.io/v1beta1'].freeze
+
+      def core_clients
+        CORE_API_VERSIONS.map do |ver|
+          [ver, create_client('api', ver)]
+        end.to_h
+      end
+
+      def group_clients
+        API_GROUPS.map do |ver|
+          [ver, create_client('apis', ver)]
+        end.to_h
+      end
+
+      def create_client(base, ver)
+        url = "#{@kubernetes_url}/#{base}"
+        log.info "create client with URL: #{url} and apiVersion: #{ver}"
+        client = Kubeclient::Client.new(
+          url, ver,
           ssl_options: ssl_options,
-          auth_options: auth_options
+          auth_options: auth_options,
+          as: :parsed
```

Comment:
This should be removed, it wasn't set before for events plugin. I remember setting the result type to `parsed` on the client won't work because for some APIs (like event watch stream) we want the raw results.

Reply:
I noticed this too. Already fixed this locally. forgot to push it when created this PR 😊